wynnset/tapestry-wp
# Add PHP linter

Closes #559

* Add PHP linter using the `php-cs-fixer` CLI tool
* Update VSCode settings to recommend `php-cs-fixer` extension
* Add PHP linting to `pre-commit` hook
* Linted all `php` files inside the `tapestry` folder

**Testing**

* Install the `php-cs-fixer` CLI following the instructions from [here](https://cs.symfony.com/)
  * For MacOS users this is as simple as using homebrew: `brew install php-cs-fixer`
* Add some unnecessary whitespace to any `php` file inside the `tapestry` folder
* Try commiting the change
* The linter should run and remove the unnecessary whitespace
